xlators/mount/fuse/utils/mount.glusterfs.in

```diff
@@ -306,11 +306,7 @@ start_glusterfs ()
         cmd_line=$(echo "$cmd_line --fuse-mountopts=$fuse_mountopts");
     fi
 
-    if [ -n "${xlator_options}" ]; then
-        for xlator_option in "${xlator_options[@]}"; do
-            cmd_line=$(echo "$cmd_line --xlator-option=$xlator_option");
-        done
-    fi
+    cmd_line="${cmd_line}${xlator_options}"
```

Comment:
I tested this on `sh` version 3.2 and it works fine. I will further test it on a debian distro (they ship with relatively older sh versions).

TLDR; I think think patch should work fine as older versions of sh struggle when dealing with `()` string interpolations. `{}` should be valid as `"$var"` expands correctly.

Regards

Reply:
@black-dragon74 could you please test your patch along with your patch on all the three debian distros and update your findings